docker/nginx/manage_images.sh

```diff
@@ -9,44 +9,30 @@ TASK=${1:-BUILD}
 echo "*** Task is $TASK"
 
 export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-eu-west-1}
+export CONFIG_BUCKET=${CONFIG_BUCKET:-platform-infra}
 
-# All out nginx containers have a dedicated repository in ECR.  We need the
-# URI for pushing the containers.
-ECR_URI=$(
-  aws ecr describe-repositories --repository-name uk.ac.wellcome/nginx | \
-  jq -r '.repositories[0].repositoryUri')
-echo "*** ECR_URI is $ECR_URI"
-
-if [[ "$ECR_URI" == "" ]]
-then
-  echo "*** Failed to read ECR repo information" >&2
-  exit 1
-fi
+# Root of the repository
+ROOT=$(git rev-parse --show-toplevel)
 
 # Directory name of the script itself
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd "$DIR"
 
-# Log in to ECR so that we can do 'docker push'
-if [[ "$TASK" == "DEPLOY" ]]
-then
-  $(aws ecr get-login)
-fi
-
 for conf_file in *.nginx.conf
 do
   variant="$(echo "$conf_file" | tr '.' ' ' | awk '{print $1}')"
   echo "*** Building nginx image for $variant..."
 
   # Construct the tag used for the image
-  TAG="$ECR_URI:$variant-$(git rev-parse HEAD)"
+  TAG="nginx:$variant-$(git rev-parse HEAD)"
   echo "*** Image will be tagged $TAG"
 
   # Actually build the image
   docker build --build-arg conf_file="$conf_file" --tag "$TAG" .
 
   if [[ "$TASK" == "DEPLOY" ]]
   then
-    docker push "$TAG"
+    $ROOT/scripts/deploy_docker_image.sh "uk.ac.wellcome/nginx" "$TAG" "$CONFIG_BUCKET" "nginx_$variant" "$(git rev-parse HEAD)"
```

Comment:
Maybe this should be entirely separate?

Write to a `.releases` directory in the root of the repo, and have the deploy script auto-push it for us